packages/webapp/src/pages/election/data.tsx

```diff
@@ -212,15 +212,23 @@ export const ElectionPage = () => {
                 {/* TODO: remove the true below once done */}
                 {(true || !Boolean(voteRowForLoggedInMember)) && (
```

Comment:
Are you still working on this or should the `true` go away?

Reply:
debatable at this point, especially since the assumption is test data, but we can always mod fixtures locally to actuate this. So... removed. thx